### PR TITLE
feat(content/pages/get-involved.mdx, content/events/*): timeline, events

### PR DESCRIPTION
**Pull request by lloydchang**
align with:
* https://github.com/gitops-working-group/gitops-working-group/pull/175
* https://github.com/cncf/tag-app-delivery/pull/148

### CODE REVIEW

**Comment by lloydchang** [comment]
@todaywasawesome Thank you for the feedback 🙂

**TL;DR:** If we should not do this in the web site, then I'm okay with the decision because I simply don't have the know-how nor time availability to learn how to implement the asks, "do this dynamically" and "expose a way to have people goto the recordings for past events".  [Gatsby plugin: MDX](https://www.gatsbyjs.com/plugins/gatsby-plugin-mdx/) isn't my forte at all.

---

My apologies if my intent and prioritized concerns were unclear earlier.

My primary and secondary concerns are to update README.md files at 1. https://github.com/cncf/tag-app-delivery/pull/148 and 2. https://github.com/gitops-working-group/gitops-working-group/pull/175

The website update is a tertiary concern to me.

Since the README.md files' content align with the web site's get-involved.mdx content, I updated get-involved.mdx here 3. https://github.com/open-gitops/website/pull/44

I believe _existing_ and _newer_ Application Delivery Technical Advisory Group and GitOps Working Group members are more likely to view README.md files in GitHub than the https://opengitops.dev web site's Get involved page.

For example, I see @RedbackThomson's [request](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.rzsq1gtvoq5f):
> [(Nick) Aggregated resource for suggested talks/workshops/demos](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.rzsq1gtvoq5f)

I hope the links in the GitHub pull requests https://github.com/cncf/tag-app-delivery/pull/148, https://github.com/gitops-working-group/gitops-working-group/pull/175, https://github.com/open-gitops/website/pull/44 to the existing videos are helpful for @RedbackThomson and other members seeking aggregated resource(s).

@todaywasawesome The existing structure of get-involved.mdx had hard coded the timeline — Based on GitHub's "Blame" view of historical changes to [get-involved.mdx](https://github.com/open-gitops/website/blame/main/content/pages/get-involved.mdx), you may need to ask @thiagosolcia or refresh your memory as to why the existing structure is hardcoded in the first place.  I don't know why.

Thank you @todaywasawesome, @RedbackThomson, @thiagosolcia for your time 🙂

**Comment by lloydchang** [comment]
@todaywasawesome I removed the timeline as we discussed.  Please review at your earliest convenience.  Thanks!
